pkg/apis/apm/v1/apmserver_types.go

```diff
@@ -40,6 +40,11 @@ type ApmServerSpec struct {
 	// SecureSettings is a list of references to Kubernetes secrets containing sensitive configuration options for APM Server.
 	// See: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-apm-server.html#k8s-apm-secure-settings
 	SecureSettings []commonv1.SecretSource `json:"secureSettings,omitempty"`
+
+	// ServiceAccountName is used to check access from the current resource to a resource (eg. Elasticsearch) in a different namespace.
+	// Can only be used if ECK is configured for checking references RBAC.
```

Comment:
This wording strikes me as a bit odd: should this say "check references with RBAC" or "enforcing RBAC on references"